build_tools/bazel_to_cmake/bazel_to_cmake_targets.py

```diff
@@ -233,37 +233,33 @@ def convert_target(target):
   if target.startswith("//llvm-external-projects/iree-dialects"):
     return _convert_iree_dialects_target(target)
 
+  if target.startswith("@"):
+    raise KeyError(f"No conversion found for target '{target}'")
```

Comment:
Let's move this above //llvm-external-projects, since it's at the end of mapping for external repositories